contracts/AccessTokenVerifier.sol

```diff
@@ -60,21 +60,11 @@ contract AccessTokenVerifier is IAccessTokenVerifier, KeyInfrastructure {
         return keccak256(abi.encode(TOKEN_TYPEHASH, token.expiry, hash(token.functionCall)));
     }
 
-    function verifySignerOf(
-        AccessToken calldata token,
-        uint8 v,
-        bytes32 r,
-        bytes32 s
-    ) external view returns (address) {
+    function verifySignerOf(AccessToken calldata token, uint8 v, bytes32 r, bytes32 s) external view returns (address) {
```

Comment:
Maybe could you add your prettier rules to the project? I remember it was me who added this and my prettier did the other notation of new line per param, but yours do the opposite;

Obv not blocking, just nice to have

Reply:
I'm good with running with yours, I honestly though mine was using the prettier config that was already in the project but if you're saying that then probably it isn't.

What's the significance of the `.prettierrc.yaml` file in this case? I'm not really a prettier 'power user', so I don't really know how best to use it xD

Reply:
I think then in this case your vscode is using the prettier rules from your global config or from another project if you opened it from another repo. afaik prettierrc.yaml is exactly for that. If you didn't purposefully changed yours then it's fine, it's always a rabbit hole to go into that